drivers/modem/ublox-sara-n310.c

```diff
@@ -491,9 +553,9 @@ static int create_socket(struct modem_socket *sock, const struct sockaddr *addr)
 		snprintk(buf, sizeof(buf), "AT+USOCR=%d", proto);
 	}
 
-	ret = modem_cmd_send(&mdata.context.iface, &mdata.context.cmd_handler,
-			     &cmd, 1U, buf, &mdata.sem_response,
-			     MDM_CMD_TIMEOUT);
+	ret = send_at_command(&mdata.context.iface, &mdata.context.cmd_handler,
```

Comment:
why did you change the function? are there specific pros/cons for the change, and/or consequences?

Reply:
> why did you change the function? are there specific pros/cons for the change, and/or consequences?

This was because I've paired checking if the module is awake and sending an AT command into the send_at_command function. It saved me from having to repeat myself (check if module is awake) every time I wanted to use modem_cmd_send.